dev/com.ibm.ws.transport.http2_fat/test-applications/H2FATDriver.war/src/http2/test/driver/war/servlets/ContinuationFrameTests.java

```diff
@@ -181,7 +181,7 @@ public void testHeaderTokenSizeExceeded(HttpServletRequest request, HttpServletR
         Http2Client h2Client = getDefaultH2Client(request, response, blockUntilConnectionIsDone);
 
         byte[] debugData = "Headers on stream: 3 exceed limits configured for the server.".getBytes();
-        FrameGoAway errorFrame = new FrameGoAway(0, debugData, COMPRESSION_ERROR, 1, false);
+        FrameGoAway errorFrame = new FrameGoAway(0, debugData, COMPRESSION_ERROR, USING_NETTY ? 2147483647 : 1, false);
```

Comment:
See if we can add this as a variable for Netty instead of the number since it's shared between different tests